<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>781</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filename">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cd">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_toparentfolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上一级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lstmode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit_search">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>退出搜索</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
